### ISSUE TITLE
Easy fix - Documentation Link Broken

### ISSUE THREAD

**Issue by ataviztik** [blizzhackers/kolbot#164]
When a character configuration file is created, lines 483-486 reference this list of skills: http://pastebin.com/baShRwWM

This is a dead link, and should probably be moved to one on https://gist.github.com/ or within this repo.

This is the list I have been referencing: https://gist.github.com/ogryzek/30b826420aacd8c1c08f1a4e94b1352f

I've searched the post compiled codebase and don't see these lines, so I'll keep digging, but whichever file is creating the .js character configs should have this in it somewhere.

**Comment by Sylvm** [comment]
This is the correct place for the list : https://github.com/blizzhackers/kolbot/blob/master/d2bs/kolbot/sdk/skills.txt

**Comment by ataviztik** [comment]
Nice, so there is a list on this repo. I did some looking, and the D2BS executable is what creates the character config files, if I'm understanding the process correctly. That would mean that only someone with access to the pre-compiled program can edit this list into it. 

tl;dr 
replace http://pastebin.com/baShRwWM 
with https://github.com/blizzhackers/kolbot/blob/master/d2bs/kolbot/sdk/skills.txt
in D2BS

**Comment by ataviztik** [comment]
It looks like the character files are generated by copying the base character files, ie d2bs/kolbot/libs/config. That means this link is wrong inside each of those, and doesn't need to be changed in the d2bs executable file. I'll go ahead and make a pull request.

**Comment by ataviztik** [comment]
welp, I lied, the base class files have the following:
_BaseConfigFile.js --> * Skills MUST be POSITIVE numbers. For reference see ...\kolbot\sdk\skills.txt

Yet, the auto generated character file contains:
 * Skills MUST be POSITIVE numbers. For reference see http://pastebin.com/baShRwWM

Looks like someone with deeper access to the exe will have to look into this one...

**Comment by DarkHorseDre** [comment]
This was updated in bh repo - currently states: `skills MUST be POSITIVE numbers. For reference see ...\kolbot\sdk\skills.txt`

**Comment by mf022** [comment]
> _BaseConfigFile.js --> * Skills MUST be POSITIVE numbers. For reference see ...\kolbot\sdk\skills.txt
> 
> Yet, the auto generated character file contains:
> 
>     * Skills MUST be POSITIVE numbers. For reference see http://pastebin.com/baShRwWM

long time ago, I wanted to rebuild and simplify the characters configuration files, keeping the different options only in the _BaseConfigFile.js
